### PR TITLE
chore(approve): Add service-registry to auto-approve list

### PR DESCRIPTION
**Pull request by brian-lou**

### CODE REVIEW

**Review by mwarkentin — APPROVED** [review]

File: .github/workflows/auto-approve.yml [hunk 1]
```diff
@@ -27,6 +27,7 @@ jobs:
         startsWith(github.event.issue.title, 'publish: getsentry/sentry-kafka-schemas@') ||
         startsWith(github.event.issue.title, 'publish: getsentry/sentry-protos@') ||
         startsWith(github.event.issue.title, 'publish: getsentry/sentry-redis-tools@') ||
+        startsWith(github.event.issue.title, 'publish: getsentry/service-registry@') ||
```
Comment (asottile-sentry) [inline .github/workflows/auto-approve.yml:30]: repo doesn't have any CI validation included in branch protection

Comment (brian-lou) [reply .github/workflows/auto-approve.yml:30]: @asottile-sentry Added status check rule: https://github.com/getsentry/service-registry/settings/branch_protection_rules/54806613

Comment (asottile-sentry) [reply .github/workflows/auto-approve.yml:30]: Screenshot 2024-10-01 at 4 20 09 PM

hmm I'm not seeing them

Comment (brian-lou) [reply .github/workflows/auto-approve.yml:30]: Screenshot 2024-10-01 at 1 33 47 PM
@asottile-sentry figured it out, the UI was just difficult to use & didn't list them

**Review by asottile-sentry — APPROVED** [review]

**Pull request merged**